src/main/java/info/novatec/camunda/migrator/ProcessVersion.java

```diff
@@ -11,6 +11,9 @@
     private final int patchVersion;
 
     public static ProcessVersion fromString(String versionString) {
```

Comment:
wouldn't it be nicer to return an `Optional<ProcessVersion>` here? Seems better to me than mapping null to 0.0.0.

Reply:
Agreed! Refactored the whole thing to use an Optional instead of Version 0.0.0.